webspicy/actions/mv.yml

```diff
@@ -0,0 +1,102 @@
+---
+  name: |-
+    Actions: mv
+
+  url: |-
+    /simplest/
+
+  services:
+  - method: |-
+      POST
+
+    description: |-
+      Allows renaming an existing directory or file
+
+    preconditions: |-
+      - The directory simplest/old should exist
+      - The file simplest/old.txt should exist
+
+    postconditions: |-
+      - The directory simplest/old must not exist
```

Comment:
suggestion: split this one test suite in two test files:

- one for the mv of files
- one for the mv of folders

Each of them only including the pre/post conditions that are relevant to the use case